src/MobileStepper/MobileStepper.js

```diff
@@ -0,0 +1,165 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import withStyles from '../styles/withStyles';
+import Paper from '../Paper';
+import Button from '../Button';
+import KeyboardArrowLeft from '../svg-icons/keyboard-arrow-left';
+import KeyboardArrowRight from '../svg-icons/keyboard-arrow-right';
+import { LinearProgress } from '../Progress';
+
+export const styleSheet = createStyleSheet('MuiMobileStepper', theme => ({
+  mobileStepper: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    width: '100%',
+    position: 'fixed',
+    bottom: 0,
+    left: 0,
+    zIndex: theme.zIndex.mobileStepper,
```

Comment:
Hum, I would rather not have any positioning property in our low-level components. One might position his element using flexbox only and not fixed positioning. (that's what we do, that would conflict). Maybe we could add a fix position option?  (Would also help with the demos!) I'm tempted to say that de default option should not be fixed.